noir-projects/noir-contracts/contracts/counter_contract/src/main.nr

```diff
@@ -155,13 +155,21 @@ pub contract Counter {
 
         let old_note = notes.get(0);
 
-        env.private().get_block_header().prove_note_validity(old_note, &mut env.private());
+        env.private().get_block_header().prove_note_validity(
+            old_note,
+            owner_slot,
```

Comment:
Perhaps we should consistently say `_storage_slot`, instead of `_slot`?

Reply:
Addressed in https://github.com/AztecProtocol/aztec-packages/pull/11942/commits/f4982091317e4c1570cca395a81ee91d4ee1517f

Reply:
Also not on this repo, perhaps you forgot to push